qw5q_platinum/platform.py

```diff
@@ -0,0 +1,74 @@
+import pathlib
+
+from qibolab.channels import Channel, ChannelMap
+from qibolab.instruments.qm import Octave, OPXplus, QMController
+from qibolab.instruments.rohde_schwarz import SGS100A
+from qibolab.kernels import Kernels
+from qibolab.platform import Platform
+from qibolab.serialize import (
+    load_instrument_settings,
+    load_qubits,
+    load_runcard,
+    load_settings,
+)
+
+FOLDER = pathlib.Path(__file__).parent
+
+
+def create():
```

Comment:
I think this not controlled by qblox cluster, right? Is QM

Reply:
Exactly, you can read it from the imports:
https://github.com/qiboteam/qibolab_platforms_qrc/blob/4a356902fff4f6a416f8b1072d8db19c5879c8dc/qw5q_platinum/platform.py#L4

Reply:
yes I know, I was just being nice :) In the docstring says:

    """QuantWare 5q-chip controlled using qblox cluster.
    Args:
        runcard_path (str): Path to the runcard file.
    """
    
    So, probably we can change it

Reply:
Indeed, the docstring was wrong, thanks @DavidSarlle. I fixed this in be0a77e6e0f205a61d84e8617b7d7ebc32d33b4f and added some details about the current calibration.